### PR TITLE
Config clean ups.

### PR DESCRIPTION
**Pull request by Harwayne**
Config clean ups.

### CODE REVIEW

File: config/201-addressable-resolvers-clusterrole.yaml [hunk 1]
```diff
@@ -12,56 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Use this aggregated ClusterRole when you need readonly access to "Addressables".
-
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: addressable-resolver
```
Comment (Harwayne) [inline config/201-addressable-resolvers-clusterrole.yaml]: I removed all of these because they duplicate what is already in https://github.com/knative/eventing/blob/master/config/200-addressable-resolvers-clusterrole.yaml. Is there a reason we would also want them defined here?

File: config/400-webhook-service.yaml [hunk 2]
```diff
@@ -21,8 +21,8 @@ metadata:
   name: webhook
   namespace: cloud-run-events
 spec:
+  selector:
```
Comment (Harwayne) [inline config/400-webhook-service.yaml]: Switched the order to match `400-controller-service.yaml`.

File: config/500-controller.yaml [hunk 3]
```diff
@@ -28,6 +28,7 @@ spec:
     metadata:
       labels:
         control-plane: cloud-run-events-controller
+        events.cloud.run/release: devel
```
Comment (Harwayne) [inline config/500-controller.yaml]: To match `500-webhook.yaml`.

File: config/500-webhook.yaml [hunk 4]
```diff
@@ -24,12 +24,12 @@ spec:
   selector:
     matchLabels:
       app: webhook
```
Comment (Harwayne) [inline config/500-webhook.yaml]: Is this label needed or wanted?

File: config/201-addressable-resolvers-clusterrole.yaml (hunk 1 above)
Comment (n3wscott) [reply config/201-addressable-resolvers-clusterrole.yaml]: I need these here. This repo does not dep on eventing to be installed for the pubsub parts.

File: config/201-channelable-manipulator-clusterrole.yaml [hunk 5]
```diff
@@ -12,12 +12,16 @@
...
+# on "Channelables". Is is defined in knative/eventing:
+# https://github.com/knative/eventing/blob/master/config/200-channelable-manipulator-clusterrole.yaml
+
+---
 
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: pubsub-channel-addressable-resolver
+  name: pubsub-channel-channelable-manipulator
```
Comment (n3wscott) [inline config/201-channelable-manipulator-clusterrole.yaml]: no no, this is just addressable-resolver.

File: config/500-webhook.yaml (hunk 4 above)
Comment (n3wscott) [reply config/500-webhook.yaml]: yes: https://github.com/knative/serving/blob/master/config/webhook.yaml#L22-L36

File: config/500-webhook.yaml [hunk 6]
```diff
@@ -24,12 +24,12 @@ spec:
...
-      role: webhook
+      role: cloud-run-events-webhook
```
Comment (n3wscott) [inline config/500-webhook.yaml]: why this change? just leave it `webhook`

File: config/500-webhook.yaml [hunk 7]
```diff
@@ -24,12 +24,12 @@ spec:
...
   template:
     metadata:
       labels:
         app: webhook
-        role: webhook
+        role: cloud-run-events-webhook
```
Comment (n3wscott) [inline config/500-webhook.yaml]: same: leave as `webhook`

File: config/400-webhook-service.yaml [hunk 8]
```diff
@@ -21,8 +21,8 @@ metadata:
...
+    role: cloud-run-events-webhook
```
Comment (n3wscott) [inline config/400-webhook-service.yaml]: `webhook`

**Comment by n3wscott** [comment]
I don't really think most of these changes should happen just because I am attempting to let cloud run pubsub to be useful without eventing installed and the structure is suppose to match serving as much as possible.

File: config/201-addressable-resolvers-clusterrole.yaml (hunk 1 above)
Comment (Harwayne) [reply config/201-addressable-resolvers-clusterrole.yaml]: Can we move these out of `config/`, so that it is an optional install instruction, rather than by default overwriting eventing's ClusterRoles? Something like `config/without-eventing/201-addressable-resolvers-clusterrole.yaml`.

File: config/201-channelable-manipulator-clusterrole.yaml (hunk 5 above)
Comment (Harwayne) [reply config/201-channelable-manipulator-clusterrole.yaml]: Why does it include create, update, and patch?

File: config/400-webhook-service.yaml (hunk 8 above)
Comment (Harwayne) [reply config/400-webhook-service.yaml]: 400-controller.yaml uses the [control-plane: cloud-run-events-controller](https://github.com/GoogleCloudPlatform/cloud-run-events/blob/8da14eb2bc52d58c256a884e11205645dcc91e66/config/400-controller-service.yaml#L22), I would like standardize on one style. My preference is to prefix with `cloud-run-events-`, but could be convinced to omit it.

Although, now that I look again, I should change the name of the label to match the controller too.

File: config/500-webhook.yaml (hunk 6 above)
Comment (Harwayne) [reply config/500-webhook.yaml]: Same reasoning as https://github.com/GoogleCloudPlatform/cloud-run-events/pull/78/files#r299373680, to match the controller's `control-plane: cloud-run-events-controller`.

File: config/500-webhook.yaml (hunk 4 above)
Comment (Harwayne) [reply config/500-webhook.yaml]: The pair of labels goes all the way back to the [initial commit](https://github.com/knative/serving/blob/7f5da0b7438f562ea4a45c7021a674e2f90a5ddf/webhook.yaml#L25-L26), but that doesn't answer why it is there. Does anything look at the `app` label and do something special with it?

File: config/201-addressable-resolvers-clusterrole.yaml (hunk 1 above)
Comment (n3wscott) [reply config/201-addressable-resolvers-clusterrole.yaml]: move sounds ok, how about `config/addressable`

File: config/500-webhook.yaml (hunk 4 above)
Comment (n3wscott) [reply config/500-webhook.yaml]: I think that is how the rp and the pods know about eachother?

File: config/201-channelable-manipulator-clusterrole.yaml (hunk 5 above)
Comment (n3wscott) [reply config/201-channelable-manipulator-clusterrole.yaml]: ah, whoops, yeah my bad. you are right. thanks

File: config/400-webhook-service.yaml (hunk 8 above)
Comment (n3wscott) [reply config/400-webhook-service.yaml]: I want just `webhook`.

**Comment by vaikas-google** [comment]
@n3wscott @Harwayne what's the plan here? Should we close this, or get it rebased?

**Comment by Harwayne** [comment]
/close

At this point it is hopelessly out of date, and if we want some of these changes, they should be made in a new PR.
